tests/dist/chats.spec.dev.js

```diff
@@ -50,24 +54,25 @@ describe('chat resolvers', function () {
       createDirectChatResponse,
       createDirectChatData;
     return regeneratorRuntime.async(function _callee2$(_context2) {
-      while (1) {
+      const condition = 1;
+      while (condition) {
         switch ((_context2.prev = _context2.next)) {
           case 0:
             _context2.next = 2;
             return regeneratorRuntime.awrap(
-            axios.post(
-              URL,
-              {
-                query:
+              axios.post(
+                URL,
+                {
+                  query:
                     '\n                mutation {\n                    createOrganization(data: {\n                        name:"test org"\n                        description:"test description"\n                        isPublic: true\n                        visibleInSearch: true\n                        }) {\n                            _id\n                        }\n                }\n                  ',
```

Comment:
You may want to consider using a template string (``) , to allow for this query to be more readable